adapters/BasicAdapter/src/main/kotlin/me/gabber235/typewriter/entries/event/BlockBreakEventEntry.kt

```diff
@@ -36,6 +37,8 @@ class BlockBreakEventEntry(
     val location: Optional<Location> = Optional.empty(),
     @Help("The item the player must be holding when the block is broken.")
     val itemInHand: Item = Item.Empty,
+    @Help("Check if a block that can age (like crops, carrots, potatoes, cocoas...) has fully grown.")
+    val ageableHasGrown : Optional<Boolean> = Optional.empty(),
```

Comment:
Instead of having this be a Boolean maybe make this a Int like minimum age. It allows for more configuration